### DISCUSSION TITLE
Gdal has failed install

### DISCUSSION THREAD

**Discussion by maharikundira** [iiasa/CWatM#171] [General]
![python image](https://github.com/user-attachments/assets/3b49ad7d-7517-4104-b954-2dd4337bad87)
![Gdal Image](https://github.com/user-attachments/assets/faa20bdd-1fbf-4c76-92fa-0ccb3c4b0fb1)

above are the images of my issue. i believe they will open

regards 
Margaret

**Comment by mikhailsmilovic** [comment]
Guten Morgen @maharikundira

What version of python is installed?
In the terminal, type
python
then enter

**Reply by maharikundira** [reply]
Dear Mikhail,
Thank you for the quick response.
installed is Python 3.13 "C:\Users\hp\Downloads\python-3.13.0-amd64.exe"
additionally, i did not see the Pfolder. i thought it would reflect after successfully installing Python.

regards
margaret

**Reply by mikhailsmilovic** [reply]
Guten Morgen Margaret,

Glad to help! In your image of the GDAL version you selected to download, what version of python is it intended for?

**Reply by maharikundira** [reply]
Guten Morgen Mikhail.

when i asked google to download Python, it brought Python 3.13 and i thought it was ok. apologies  am new to this field but i want to learn.

regards

**Comment by mikhailsmilovic** [comment]
Guten Morgen, python 3.13 is fine, but the GDAL you downloaded is for python 3.12 😊
The CWatM Level A playlist will guide you through these small details. 

[This is the video for the installation, including GDAL](https://youtu.be/gW6AUELHvu0?si=cxK7LwTfhC-esxVm)

[Here is the link to the whole playlist](https://youtube.com/playlist?list=PLyT8dd_rWLawkTUNNXIIhoTxm_fGkI9Vf&si=GwZMittAJ3yv8JBP)
